src/zenml/cli/utils.py

```diff
@@ -1388,15 +1419,28 @@ def inner_decorator(func: F) -> F:
         data_type_descriptors = set()
         for k, v in filter_model.__fields__.items():
             if k not in filter_model.CLI_EXCLUDE_FIELDS:
-                options.append(
-                    click.option(
-                        f"--{k}",
-                        type=str,
-                        default=v.default,
-                        required=False,
-                        help=create_filter_help_text(filter_model, k),
+                # if k is metadata, add option with multiple values
+                if k == "metadata_values":
```

Comment:
Can we make this more generic? I envision something that adds a `multiple=True` option in case the filter model property is a collection of some sort (Dict, List)

Reply:
Can we also remove this filtering  logic here in the CLI then?